flexmeasures/data/models/charts/defaults.py

```diff
@@ -8,6 +8,7 @@
 WIDTH = 600
 REDUCED_HEIGHT = REDUCED_WIDTH = 60
 SELECTOR_COLOR = "darkred"
+SOURCE_TITLE = "Source"
```

Comment:
I believe a better name would be `SOURCE_SELECTOR_TITLE`, as it's not the title of one or more sources ...